ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -427,7 +427,32 @@ aws|aws-arm64|aws-usgov)
     ;;
 esac
 
-sed -i '/^  channel:/d' "${dir}/manifests/cvo-overrides.yaml"
+function setup-cvo() {
+    local payload_version
+    payload_version="$(oc adm release info "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE}" -o "jsonpath={.metadata.version}")"
+    echo "Release payload version: ${payload_version}"
+    local channel
+    channel="$(grep -E --only-matching '(stable|eus|fast|candidate)-4.[0-9]+' "${dir}/manifests/cvo-overrides.yaml")"
+    echo "Original channel from CVO manifest: ${channel}"
+    local candidate_channel
+    # WTF THIS SHOULD WORK candidate_channel="${channel//+(stable|eus|fast)/candidate}"
+    candidate_channel="$(echo "${channel}" | sed -E 's/(stable|eus|fast)/candidate/')"
+    echo "Matching candidate channel: ${candidate_channel}"
+
+    if curl --silent "https://api.integration.openshift.com/api/upgrades_info/v1/graph?channel=$candidate_channel" | grep --quiet '"version":"'"$payload_version"'"'; then
```

Comment:
did we want to set `arch` too?  Or are we comfortable assuming that if the version is in channels for `amd64` it will exist for whatever arch this release happens to be?

Reply:
I think we should do that, but I have trouble finding a good source for that string. It seems that `multi` payloads have a `.metadata.metadata['release.openshift.io/architecture']` key but other do not, but there's also `.config.architecture`  which is correct everywhere except multi...:

```go
$ for arch in x86_64 ppc64le aarch64 multi
      set meta (oc adm release info quay.io/openshift-release-dev/ocp-release:4.13.6-$arch -o "jsonpath={.metadata.metadata.release\.openshift\.io/architecture}")
      echo (string pad -r -w 8 $arch)": .metadata.metadata['release.openshift.io/architecture']=$meta"
      set cfg (oc adm release info quay.io/openshift-release-dev/ocp-release:4.13.6-$arch -o "jsonpath={.config.architecture}")
      echo (string pad -r -w 8 $arch)": .config.architecture=$cfg"
  end

x86_64  : .metadata.metadata['release.openshift.io/architecture']=
x86_64  : .config.architecture=amd64
ppc64le : .metadata.metadata['release.openshift.io/architecture']=
ppc64le : .config.architecture=ppc64le
aarch64 : .metadata.metadata['release.openshift.io/architecture']=
aarch64 : .config.architecture=arm64
multi   : .metadata.metadata['release.openshift.io/architecture']=multi
multi   : .config.architecture=amd64
```

I guess OSUS needs some data when it scrapes, I guess I should look at what it does...

Reply:
Done